src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -1,43 +1,40 @@
 <Project>
   <PropertyGroup>
     <AsnXml />
+    <_AsnXmlDiffCmd>%24%28command -v diff%29 -q -a</_AsnXmlDiffCmd>
+    <_AsnXmlDiffCmd Condition="$([MSBuild]::IsOsPlatform('windows')) == 'true'">$(SystemRoot)\System32\fc.exe /a</_AsnXmlDiffCmd>
+    <_AsnIntermediatePath>$([MSBuild]::NormalizeDirectory('$(IntermediateOutputPath)', 'asnxml'))</_AsnIntermediatePath>
```

Comment:
`diff` was getting cranky being passed Windows directory separator when using `$(IntermediateOutputPath)asnxml\%(filename).cs`. Instead I made a variable here which will ensure there is a trailing slash that is appropriate for the current OS.